foundry.toml

```diff
@@ -0,0 +1,14 @@
+[default]
+src = 'src'
+out = 'out'
+libs = ['lib']
+remappings = [
+    'ds-test/=lib/ds-test/src/',
+    'openzeppelin-contracts/=lib/openzeppelin-contracts/contracts/',
+    '@openzeppelin/=lib/openzeppelin-contracts/',
+    'uma/=lib/protocol/packages/core/contracts/',
+    'yamV3/=lib/yamV3/contracts/',
+]
```

Comment:
we need to use `remappings.txt` instead to satisfy both dapptools and forge in the same time, so this part can be removed completely